third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -271,6 +271,11 @@ pub static EXPERIMENTS: Lazy<BTreeMap<String, Experiment>> = Lazy::new(|| {
                 .to_string(),
             default: Given(false),
         },
+        Experiment {
```

Comment:
Could we:
1. Add a test folder under `third_party/move/move-compiler-v2/tests/`, where we add some tests (i.e., move files with errors and warnings, you can use some of the existing tests in other folders that have errors/warnings for this purpose)
2. Create a `TestConfig` for this test folder in `third_party/move/move-compiler-v2/tests/testsuite.rs` where we selectively enable this experiment for that folder, use `UB=1 cargo test` under `move-compiler-v2` package to generate the `.exp` files (the `UB=1` updates baselines).

Then, the `.exp` files will serve to showcase that this experiment flag works.